docs/plugin.md

```diff
@@ -760,4 +762,4 @@ More plugins are available as reference on poooi(https://github.com/poooi)
 
 
 ## Community
-Welcome to become developer for poi. You may post your questions and ideas as github issues, or join our [telegram group](https://telegram.me/joinchat/AoMUpkCr6B8uH7EUewq6eQ) for discussions.
\ No newline at end of file
+Welcome to become developer for poi. You may post your questions and ideas as github issues, or join our [telegram group](https://telegram.me/joinchat/AoMUpkCr6B8uH7EUewq6eQ) for discussions.
```

Comment:
What is the difference? line feed?

Reply:
sorry, I don't know, maybe because of the text editor